index.js

```diff
@@ -291,9 +299,10 @@ export default class TextMarquee extends PureComponent {
           horizontal
           scrollEnabled={scroll ? !this.state.contentFits : false}
           scrollEventThrottle={16}
-          onScroll={this.resetScroll}
+          onScrollBeginDrag={this.scrollBegin}
+          onScrollEndDrag={this.scrollEnd}
```

Comment:
Removed `onScroll={this.resetScroll}` and split into `onScrollBeginDrag={this.scrollBegin}` and `onScrollEndDrag={this.scrollEnd}`.

The reason is that the ScrollView onScroll event is constantly triggered when `I18nManager.isRTL` is true, preventing the animation working properly. I'm still experiencing this issue in RN 0.62.2 See: https://github.com/facebook/react-native/search?q=ScrollView+RTL&type=Issues for more info